### ISSUE TITLE
Collecting and reporting visitor statistics

### ISSUE THREAD

**Issue by danielvartan** [ropensci-org/rotemplate#51]
Hi,

I recently moved my package to rOpenSci. All things worked as described, but the Google Analytics tracking are not showing up in the code. Is there something I miss?

The package documentation website can be found at https://docs.ropensci.org/mctq/ .

Cheers!

**Comment by jeroen** [comment]
We deliberately do not support google analytics in our template. @maelle

**Comment by danielvartan** [comment]
I see. Do you guys suggest some other solution for web analytics?

**Comment by maelle** [comment]
@jeroen should we make it possible to override that? E.g. with pkgdown dev version (to be released in the next couple of weeks)

```yaml
template:
  includes:
    in_header: <!-- inserted at the end of the head -->
  package: rotemplate
```

Would work. Now the question is, how much use of the template field does our docs building system enable?

------------------------------------------------

@danielvartan we use Matomo Analytics. What data would you like us to provide, what did you monitor with Google Analytics? Not promising a report in the very short term, but definitely interested in listing requirements & in working on it.

**Comment by jeroen** [comment]
I don't think it fits in our privacy policy to allow 3rd party trackers to be embedded under our `.ropensci.org` domains.

**Comment by maelle** [comment]
now that's a good point! :grimacing: 

So let's focus on making a list of what data package authors would like to see summarized regularly.

**Comment by danielvartan** [comment]
Hi @maelle ,

I'm interested in just some basic visitor data (e.g., frequency of unique visitors,  geographical location, most visited pages, traffic acquisition, bounce rate). The objective is to measure user engagement to try optimize the user experience.

**Comment by maelle** [comment]
Thanks @danielvartan! What time resolution were you previously using? Did you have some sort of custom report you could share? (no worries if not!)

**Comment by danielvartan** [comment]
A 1-month window usually. I don't have a report to share (sorry!).

**Comment by maelle** [comment]
So a report with data aggregated by month, correct?

Thank you!

Out of curiosity, in the past, did the data help you make a practical decision? (no worries if not!)

**Comment by danielvartan** [comment]
I can't remember any direct practical design decision I made with this data. I use mostly for insights.

For example, by knowing the geographical location of the users, I can direct or redirect my promotion efforts to some places, by choosing to promote the package in specific circles.

The aggregated data also helps me to measure the impact of the package. This is important to show, for example, to funding agencies and to assure the community itself that can benefit from using the package.

**Comment by maelle** [comment]
Re-opening as we need to sort out the reporting, and as the other issue where I track this is private.

**Comment by jeroen** [comment]
OK let's update the title then

**Comment by maelle** [comment]
I'm not forgetting this but work on this will have to happen in 2022. Thanks for your patience!
